app/src/main/java/com/example/concordia_campus_guide/Models/CalendarEvent.java

```diff
@@ -31,9 +31,10 @@ public String getStartTime() {
         return startTime;
     }
 
-    public void setStartTime(String startTime) {
-        this.startTime = startTime;
-    }
+    //TODO: to be deleted because app should not be able to change startTime
+//    public void setStartTime(String startTime) {
+//        this.startTime = startTime;
+//    }
```

Comment:
if it is not needed anymore. Can you remove it please ?

Reply:
sure